addons/medical_treatment/stringtable.xml

```diff
@@ -929,22 +929,26 @@
             <German>Fahrzeuge und Einrichtungen</German>
             <Turkish>Araçlar ve Tesisler</Turkish>
             <Spanish>Vehículos e instalaciones médicas</Spanish>
-        </Key>
+        </Key>        
```

Comment:
Looks like you accidentally put some spaces here.

Reply:
```suggestion
        </Key>
```